websockify/websocket.py

```diff
@@ -311,16 +324,10 @@ def send_frames(self, bufs=None):
         if bufs:
             for buf in bufs:
                 if self.base64:
-                    encbuf, lenhead, lentail = self.encode_hybi(buf, opcode=1, base64=True)
+                    opcode = 1
                 else:
-                    encbuf, lenhead, lentail = self.encode_hybi(buf, opcode=2, base64=False)
-
-                if self.rec:
-                    self.rec.write("%s,\n" %
-                            repr("{%s{" % tdelta
-                                + encbuf[lenhead:len(encbuf)-lentail]))
-
-                self.send_parts.append(encbuf)
+                    opcode = 2
+                encbufs = self.send_hybi(buf, opcode, base64=self.base64, record=self.rec)
```

Comment:
`send_hybi` doesn't look like it returns anything.  Also, doesn't this make the `whiel self.send_parts` below redundant?